Library/Formula/avidemux.rb

```diff
@@ -124,6 +115,7 @@ def install
           -DPLUGIN_UI=#{plugin}
           -DAVIDEMUX_LIB_DIR=#{lib}
           -DAVIDEMUX_SOURCE_DIR=#{buildpath}
+          -DCMAKE_OSX_SYSROOT=#{MacOS.sdk_path}"
```

Comment:
My thoughts at this point are mostly confused noises. I'll find some time today to spin up a Yosemite VM and poke this a bit. That error is _very_ strange.